storage.go

```diff
@@ -359,7 +428,11 @@ func (s *Storage) stat(ctx context.Context, path string, opt pairStorageStat) (o
 	o = s.newObject(true)
 	o.ID = rp
 	o.Path = path
```

Comment:
We need to trim the `Path` trailing `/`

Reply:
I’m confused about it. Is `o.Path` not the `path` passed in?

Reply:
Yes, in most time `o.Path == path`.

~~But for dir object, we add `/` after path `test` and send real API to `test/`. So after we got the information, we need to roll it back.~~

~~Maybe we could make it more clear by creating a new var: `sp := path + "/"`.~~

In order to make this behavior works as expected, we can add check in `go-integration-test`, make sure returning `path` equals to input `path`.

---

Ooooops, the path hasn't been updated. Current logic is correct.